### PR TITLE
feat(route): add Oxford Academic - Oxford University Press

### LINKED ISSUE #9155: Oxford Journal Latest Issues
### 类型

科学期刊

### 网站地址

https://academic.oup.com/journals

 e.g. https://academic.oup.com/sysbio/issue
 e.g. https://academic.oup.com/zoolinnean/issue

### 网站描述

Oxford journals is a global journal platform.

### 需要生成什么内容？

- Title
- Authors
- Abstract
- Keywords

### 额外描述

The RSS provided by the official web can't be used

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by Fatpandac**
## 该 PR 相关 Issue / Involved issue

Close #9155

## 完整路由地址 / Example for the proposed route(s)

```routes
/oup/journals/beheco
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [X] 新的路由 New Route
  - [X] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [X] 文档说明 Documentation
  - [X] 中文文档 CN
  - [X] 英文文档 EN
- [X] 全文获取 fulltext
  - [X] 使用缓存 Use Cache
- [X] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [X] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [X] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — APPROVED** [review]

File: docs/journal.md [hunk 1]
```diff
@@ -210,6 +210,14 @@ pageClass: routes
 
 <Route author="yech1990" example="/nature/highlight" path="/nature/highlight" supportScihub="1"/>
 
+## Oxford University Press
+
+### Oxford Academic
+
+#### 期刊
+
+<Route author="Fatpandac" example="/oup/journals/adaptation" path="/oup/journals/:name" :paramsDesc="['期刊名称缩写，可以在网址中得到']"/>
```
Comment (TonyRL) [inline docs/journal.md:219]:
```suggestion
<Route author="Fatpandac" example="/oup/journals/adaptation" path="/oup/journals/:name" :paramsDesc="['期刊名称缩写，可以在网址中得到']" anticrawler="1"/>
```

File: docs/en/journal.md [hunk 2]
```diff
@@ -237,6 +237,14 @@ Subscribe to the cover images of the Nature journals, and get the latest publica
 
 <RouteEn author="yech1990" example="/nature/highlight" path="/nature/highlight" supportScihub="1"/>
 
+## Oxford University Press
+
+### Oxford Academic
+
+#### Journal
+
+<Route author="Fatpandac" example="/oup/journals/adaptation" path="/oup/journals/:name" :paramsDesc="['short name for a journal, can be found in URL']"/>
```
Comment (TonyRL) [inline docs/en/journal.md:246]:
```suggestion
<Route author="Fatpandac" example="/oup/journals/adaptation" path="/oup/journals/:name" :paramsDesc="['short name for a journal, can be found in URL']" anticrawler="1"/>
```

File: lib/v2/oup/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    const response = await got(url);
+    const cookies = response.headers['set-cookie'].map((item) => item.split(';')[0]).join(';');
+    const $ = cheerio.load(response.data);
+    const list = $('div.al-article-items')
+        .map((_, item) => ({
+            title: $(item).find('a.at-articleLink').text(),
+            link: new URL($(item).find('a.at-articleLink').attr('href'), rootUrl).href,
+        }))
+        .get();
```
Comment (TonyRL) [inline lib/v2/oup/index.js:21]: Consider using the `limit` query parameter.

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/oup/index.js (hunk 3 above)
Comment (Fatpandac) [reply lib/v2/oup/index.js:21]: Have not any good happens after add limit.

**Pull request merged**